previews/primer/beta/relative_time_preview/link_with_tooltip.html.erb

```diff
@@ -0,0 +1,23 @@
+<script>
+  // When `<relative-time>`’s `title` changes, update `#tooltip-link`’s associated `<tool-tip>`.
+  new MutationObserver(function (mutationList) {
+    for (const mutation of mutationList) {
+      if (mutation.target.matches("relative-time")) {
+        const title = mutation.target.getAttribute("title")
+        const tooltipId = document.getElementById("tooltip-link")?.getAttribute("aria-describedby")
+        const tooltip = document.getElementById(tooltipId)
+        if (tooltip) {
+          tooltip.textContent = title
+        }
+      }
+    }
+  }).observe(document.body, {
+    attributeFilter: ["title"],
+    subtree: true,
+  });
+</script>
```

Comment:
`<relative-time>` is rendered on the client, so it can (and does) set its `title` attribute’s value to local time.

Moving to Primer View Components’ Tooltip improves accessibility, but because PVC is server-rendered, it uses server (or UTC) times.

This JavaScript copies `<relative-time>`’s local time into PVC’s Tooltip. As written, it relies on the fact that `<relative-time>` still updates `title` (because the `no-title` update hasn’t shipped). If we like this approach, we’ll want to make `<relative-time>` write the localized date to a `data-` attribute instead, then update lines 15 and 6 here, accordingly.

Reply:
Update: https://github.com/primer/view_components/pull/2872/commits/b8dcbbcb88e4561128895a681f206809be0fb99f is a simpler event-based solution.